Capri.Web/Controllers/PromoterController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Capri.Services;
+using Capri.Web.ViewModels.Promoter;
+
+namespace Capri.Web.Controllers
+{
+    public class PromoterController : Controller
```

Comment:
Add here annotation `promoters`. You won't need to add them to every endpoint. It will just concatenate prefix from controller and endpoint :)